model/src/cmd/ModelCommandAPI.cc

```diff
@@ -0,0 +1,369 @@
+                            + std::to_string(poseComp->Data().Pitch()) + " | "
+                            + std::to_string(poseComp->Data().Yaw()) + "]";
+
+    std::cout << "Name: " << nameComp->Data() << std::endl
+              << "  - Pose: " << poseInfo << std::endl << std::endl;
+  }
+}
+
+//////////////////////////////////////////////////
+// \brief Print the model links information.
+// \param[in] _entity Entity of the model requested.
+// \param[in] _ecm ECM ready for requests.
+// \param[in] _linkName Link to be printed, if nullptr, print all links.
+void printLinks(const uint64_t _entity,
+                const ignition::gazebo::EntityComponentManager &_ecm,
+                const std::string &_linkName)
+{
+  const auto links = _ecm.EntitiesByComponents(
+  ignition::gazebo::components::ParentEntity(_entity),
+  ignition::gazebo::components::Link());
+  for(const auto &entity : links)
+  {
+    const auto parentComp =
+        _ecm.Component<ignition::gazebo::components::ParentEntity>(entity);
+
+    const auto nameComp =
+        _ecm.Component<ignition::gazebo::components::Name>(entity);
+
+    if(_linkName.length() && _linkName != nameComp->Data())
+        continue;
+
+    if (parentComp)
+    {
+      std::string parentInfo;
+      const auto parentNameComp =
+          _ecm.Component<ignition::gazebo::components::Name>(
+          parentComp->Data());
+
+      if (parentNameComp)
+      {
+        parentInfo += parentNameComp->Data() + " ";
+      }
+      parentInfo += "[" + std::to_string(parentComp->Data()) + "]";
+      std::cout << "  - Link [" << entity << "]" << std::endl
+                << "    - Name: " << nameComp->Data() << std::endl
+                << "    - Parent: " << parentInfo << std::endl;
+    }
+
+    const auto inertialComp =
+        _ecm.Component<ignition::gazebo::components::Inertial>(entity);
+
+    if (inertialComp)
+    {
+      const auto inertialMatrix =  inertialComp->Data().MassMatrix();
+      const auto massComp = inertialComp->Data().MassMatrix().Mass();
+
+      const std::string massInfo = "[" + std::to_string(massComp) + "]";
+      const std::string inertialInfo =
+            "\n        [" + std::to_string(inertialMatrix.Ixx()) + " | "
+                          + std::to_string(inertialMatrix.Ixy()) + " | "
+                          + std::to_string(inertialMatrix.Ixz()) + "]\n"
+              "        [" + std::to_string(inertialMatrix.Ixy()) + " | "
+                          + std::to_string(inertialMatrix.Iyy()) + " | "
+                          + std::to_string(inertialMatrix.Iyz()) + "]\n"
+              "        [" + std::to_string(inertialMatrix.Ixz()) + " | "
+                          + std::to_string(inertialMatrix.Iyz()) + " | "
+                          + std::to_string(inertialMatrix.Izz()) + "]";
+      std::cout << "    - Mass: " << massInfo << std::endl
+                << "    - Inertial Matrix: " << inertialInfo << std::endl;
```

Comment:
It would be useful to also print the inertial pose

Reply:
Addressed